packages/core/src/components/Button/Button.tsx

```diff
@@ -220,7 +222,7 @@ const Button: VibeComponent<ButtonProps, unknown> & {
           [styles.leftFlat]: leftFlat,
           [styles.preventClickAnimation]: preventClickAnimation,
           [styles.noSidePadding]: noSidePadding,
-          [styles.disabled]: disabled,
+          [styles.disabled]: disabledButton,
```

Comment:
Is it ok that it's disabled style if `aria-hidden` is provided? Not a side effect?

Reply:
hmm I get what you mean. yep. let's leave it with the `disabled` only, unless something weird comes up